install_config/configuring_aws.adoc

```diff
@@ -34,9 +32,55 @@ Zone = us-east-1c <1>
 <1> This is the Availability Zone of your AWS Instance and where your EBS Volume
 resides; this information is obtained from the AWS Managment Console.
 ====
-
+ 
 [[aws-configuring-masters]]
-== Configuring Masters
+== Configuring {product-title} Masters for AWS
+
+You can set AWS configuration on your {product-title} masters two ways:
```

Comment:
"You can set the AWS configuration on your {product-title} master hosts in two ways:"